pmix/borrow.py

```diff
@@ -127,7 +128,7 @@ def borrow_cli():
         description='Grab translations from existing XLSForms'
     )
     parser.add_argument(
-        'xlsxfile', nargs='+',
+        'xlsxfiles', nargs='+',
```

Comment:
This should be `xlsxfile` because of the generated help text.

Reply:
Generated? If you could link me to that it would help. I don't see anywhere where plural wouldn't be good as a replacement: "1 or more xlsxfiles". Having it singular is confusing.

Reply:
<img width="616" alt="screenshot 2018-12-03 14 19 53" src="https://user-images.githubusercontent.com/3988950/49405191-8e0ed000-f706-11e8-8671-88f07f9a6e8a.png">

Reply:
`xlsxfile [xlsxfile ...]` is better than `xlsxfiles [xlsxfiles ...]`

Reply:
Hmm, I see. That's too bad.

Actually, I prefer named arguments. I'd prefer `--xlsxfiles` or `--source_files`, rather than positional arguments, but that could just be a preference; IDK if I can really argue over one being better than the other.

I'll change back